src/components/Tokens/TokenTable/SearchBar.tsx

```diff
@@ -25,12 +27,13 @@ const SearchInput = styled.input`
   border-radius: 12px;
   border: 1px solid ${({ theme }) => theme.surface3};
   height: 100%;
-  width: min(200px, 100%);
+  width: ${({ isExplore, isClicked }) => (isExplore ? (isClicked ? '200px' : '0') : 'min(200px, 100%)')};
   font-size: 16px;
   font-weight: 485;
   padding-left: 40px;
   color: ${({ theme }) => theme.neutral2};
   transition-duration: ${({ theme }) => theme.transition.duration.fast};
```

Comment:
with this new grow/shrink animation we should probably slow down the animation, show the preview to @bryankuuu or @infredible to get some input!